lxd/storage_volumes.go

```diff
@@ -1492,24 +1492,14 @@ func storagePoolVolumePost(d *Daemon, r *http.Request) response.Response {
 				return resp
 			}
 
-			srcPool, err := storagePools.LoadByName(s, srcPoolName)
-			if err != nil {
-				return response.SmartError(err)
-			}
-
-			if srcPool.Driver().Info().Name == "ceph" {
+			// FIXME: Should this be `if pool.Driver().Info().Remote {...}`? E.g. what about powerflex?
+			if pool.Driver().Info().Name == "ceph" {
```

Comment:
@tomponline @roosterfish I thought this line was quite suspicious. Do you know if this block is specific to the Ceph driver? Or should it be for remote drivers?

Reply:
Interesting. It doesn't look Ceph specific to me.
Did a quick search on the code base and also found https://github.com/canonical/lxd/blob/main/lxd/instance_post.go#L987. Looks all of those help moving Ceph based volumes from an offline cluster member. I agree this should be generic for remote storage drivers. 

Should we have a separate issue so we can follow up on this?

Reply:
At least I cannot see why this logic should only apply for Ceph based storage vols.

Reply:
> Interesting. It doesn't look Ceph specific to me.

Agreed, it should be changed to cover any remote pool - although it will need to be tested with powerflex.

Reply:
> Did a quick search on the code base and also found https://github.com/canonical/lxd/blob/main/lxd/instance_post.go#L987. Looks all of those help moving Ceph based volumes from an offline cluster member. I agree this should be generic for remote storage drivers.
> 
> Should we have a separate issue so we can follow up on this?

Yes that looks like it shouldnt be ceph specific either.

I think a separate issue and a search for `== "ceph"` to check for the missed items from when the powerflex driver was added will be needed here.

@roosterfish presumably that means that some functionality in the powerflex driver isn't working correctly at this time?

Reply:
@tomponline for this specific scenario that is possible yes. I'll have a look into this.